content/en/opentelemetry/guide/migrate/migrate_operation_names.md

```diff
@@ -12,17 +12,54 @@ further_reading:
 
 When using OpenTelemetry with Datadog, you might see unclear or lengthy operation names in your traces, and some traces might not appear in your service pages. This happens because of missing mappings between OpenTelemetry SDK information and Datadog operation names, which are span attributes that classify [entry points into a service][1].
 
-Datadog now provides new operation name mappings that improve trace visibility in the service page. This feature requires opt-in configuration now but will become the default in the near future.
+Datadog has introduced new logic for generating operation names for OpenTelemetry traces, controlled by the `enable_operation_and_resource_name_logic_v2` feature flag. This new logic improves trace visibility in service pages and standardizes operation naming according to the rules outlined below.
 
-For example, a span that has the old operation name `go.opentelemetry.io_contrib_instrumentation_net_http_otelhttp.server` is named `http.server.request` with the new logic.
-
-<div class="alert alert-warning">
-Datadog strongly recommends migrating to the new mappings as soon as possible before they become the default.<br>If you have monitors or dashboards observing operation names, they will be affected by the change. Please update them to observe the new conventions.
+<div class="alert alert-danger">
+<strong>Breaking Change:</strong> When this new logic is active (either by opting-in or future default), it is a breaking change for monitors or dashboards that reference operation names based on the old conventions. You must update your monitors and dashboards to use the new naming conventions described in <a href="#new-mapping-logic">New mapping logic</a>. If you cannot update them yet, you can <a href="#disabling-the-new-logic-opt-out">opt out</a> .
 </div>
 
+## Default rollout schedule
+
+The `enable_operation_and_resource_name_logic_v2` feature flag controls this new logic. It is enabled by default starting in the following versions:
+
+- **Datadog Distribution of OpenTelemetry (DDOT) Collector**: Datadog Agent v7.65+
+- **OpenTelemetry Collector**: OTel Collector v0.126.0+
+- **OTel to Datadog Agent (OTLP)**: Datadog Agent v7.66+
+
+Prior to these releases, you must explicitly [enable the logic](#enabling-the-new-logic-opt-in).
```

Comment:
```suggestion
If you are using an earlier version and want to use the new logic without upgrading, you can [explicitly enable the new logic](#enabling-the-new-logic-opt-in).
```

As-is, this sentence means "Prior to these releases, you must explicitly enable the logic, in all circumstances" but I think that you actually mean "Prior to these releases, if you want to use the new logic, you must explicitly enable the new logic."